server/portal/apps/search/tasks.py

```diff
@@ -26,17 +26,21 @@ def agave_indexer(self, systemId, filePath='/', recurse=True, update_pems=False,
     if not filePath.startswith('/'):
         filePath = '/' + filePath
 
+    if next(sys for sys in settings.PORTAL_DATAFILES_STORAGE_SYSTEMS
+        if sys['name'] == 'Shared Workspaces' and sys['hideSearchBar'] == True
```

Comment:
maybe use `sys['scheme'] == 'projects'` in case we want to change the name?
```suggestion
        if sys['scheme'] == 'projects' and sys['hideSearchBar'] == True
```